src/legacy/core_plugins/vis_default_editor/public/components/agg_params.tsx

```diff
@@ -219,35 +219,34 @@ function DefaultEditorAggParams({
       })}
 
       {params.advanced.length ? (
-        <EuiFormRow>
-          <EuiAccordion
-            id="advancedAccordion"
-            data-test-subj={`advancedParams-${agg.id}`}
-            buttonContent={i18n.translate('visDefaultEditor.advancedToggle.advancedLinkLabel', {
-              defaultMessage: 'Advanced',
-            })}
-          >
-            <EuiSpacer size="s" />
-            {params.advanced.map(param => {
-              const model = paramsState[param.aggParam.name] || {
-                touched: false,
-                valid: true,
-              };
-
-              return (
-                <DefaultEditorAggParam
-                  key={`${param.aggParam.name}${agg.type ? agg.type.name : ''}`}
-                  disabled={disabledParams && disabledParams.includes(param.aggParam.name)}
-                  formIsTouched={formIsTouched}
-                  showValidation={formIsTouched || model.touched}
-                  setAggParamValue={setAggParamValue}
-                  onChangeParamsState={onChangeParamsState}
-                  {...param}
-                />
-              );
-            })}
-          </EuiAccordion>
-        </EuiFormRow>
+        <EuiAccordion
+          className="visEditorAggParams__advanced"
```

Comment:
Just to make it easier, I'd suggest not using styles but EuiSpacer to create the margins. Then there's less cross-referencing to figure out what the classes are doing.

Reply:
Makes sense.
Updated